objects/obj_drop_select/Alarm_5.gml

```diff
@@ -1,26 +1,44 @@
-
+// This script sets up the selected forces to drop into a planet and initiates combat
 attacking=10;
 
-var add_ground;add_ground=0;        // Local Forces here
-var smin,smax,w;w=-1;smin=0;smax=0;
-
-if (add_ground=1){ships_selected+=1;remove_local=-1;
-    bikes+=l_bikes;rhinos+=l_rhinos;
-    whirls+=l_whirls;predators+=l_predators;
-    raiders+=l_raiders;speeders+=l_speeders;
+// Local Forces here
+var add_ground=0;        
```

Comment:
0 => false (if it's a boolean?)

Reply:
it has 3 values, -1, 0 and 1, so cant convert to boolean